docs/src/components/landing-page/TwitterShowcaseCards.vue

```diff
@@ -165,6 +147,11 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
+/*
+ [1], [2]: needed to create a white card behind the twitter card, this card gets shown when moving to previous carousel
```

Comment:
Numbers here should only be one per "feature", not one per line of code used: you're mapping lines to the comment, not the other way around

Here there shouldn't be `[2]`, and the line marked with `[2]` should be marked with `[1]`
Dunno if the concept is clear, I can re-explain it if needed